library/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './swarm'
```

Comment:
import `swarm-html` in the index.ts
```suggestion
import './swarm-html'
export * from './swarm'
```